sections/04_study.md

```diff
@@ -36,9 +36,16 @@ from gene expression.  Those could be divided into separate sub-sections.*
 
 ### Transcription factors and RNA-binding proteins
 
-*Existing reviews have covered some of these papers rather well and we do not
-want to repeat what has already been well-stated elsewhere.  This could
-be split into two sub-sections or kept very brief.*
+Transcription Factor and RNA-binding proteins are key components for gene regulation, making them very important to understand for higher level biological processes. While high-throughput sequencing techniques such as ChIP-seq have been able to accurately identify binding regions for DNA and RNA proteins, these experiments are both time consuming and expensive. In addition, the sequencing methods do not provide any sort of analysis on the proteins which would lead to a better understanding of the underlying process. Thus, there is a need to computationally predict and understand these binding regions de novo from sequences.
+
+####Transcription Factors
+
+Transcription Factors (TFs) are regulatory proteins which bind to certain locations on a DNA sequence and controls the rate of genetic information transcription. Chromatin immunoprecipitation and massively parallel DNA sequencing (ChIP-seq) technologies are able to identify highly likely binding sites for a certain TF, and databases such as ENCODE have provided ChIP-seq data for hundreds of different TFs across many laboratories. However, ChIP-seq experiments are expensive and time consuming. Since the data that scientists have discovered is available, in silico methods to predict binding sites are of great interest, thus eliminating the need to do new ChIP-seq experiments every time analysis is done on a new sequence. 
```

Comment:
Fixed the ChIP-seq issue, thanks.

Yeah, so I'm supposed to be summarizing TFBS and RNA binding, that's why I have the intro paragraph, but I haven't done the RNA binding subsection yet because I'm not too familiar with it. Not sure if we would still keep that intro paragraph though.